satpy/readers/abi_l1b.py

```diff
@@ -158,47 +92,6 @@ def get_dataset(self, key, info):
 
```

Comment:
Github won't let me comment any higher, but just below line 80 there should be a line remove `_Unsigned` which was done in https://github.com/pytroll/satpy/pull/838.